lcd/lcd_api.py

```diff
@@ -140,6 +140,7 @@ def putchar(self, char):
             if self.implied_newline:
                 # self.implied_newline means we advanced due to a wraparound,
                 # so if we get a newline right after that we ignore it.
+                self.implied_newline = False
                 pass
```

Comment:
Now that there is a statement inside the if, the pass should be removed

Reply:
Good point.  Removed statement and pushed update.